app/views/candidate_interface/references/retry_request/new.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :title, t('page_titles.references_retry_request') %>
-<% content_for :before_content, govuk_back_link_to %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_references_review_path) %>
```

Comment:
There's no back link on the _Request reference_ form like there is here. Seems inconsistent but it's nothing to do with this change. Do you think it's worth logging a card or raising with a UX person?

<img width="713" alt="image" src="https://user-images.githubusercontent.com/450843/120561319-37300180-c3fc-11eb-8c86-257a3e137402.png">

Reply:
I think we can just add one. I see no reason there shouldn't be one.